libraries/browser-tracker-core/src/helpers/browser_props.ts

```diff
@@ -55,5 +58,18 @@ function detectDocumentSize() {
     bodyHeight = be ? Math.max(be.offsetHeight, be.scrollHeight) : 0;
   var w = Math.max(de.clientWidth, de.offsetWidth, de.scrollWidth);
   var h = Math.max(de.clientHeight, de.offsetHeight, de.scrollHeight, bodyHeight);
-  return isNaN(w) || isNaN(h) ? '' : w + 'x' + h;
+  return isNaN(w) || isNaN(h) ? '' : w + DIMENSION_SEPARATOR + h;
+}
+
+function detectScreenResolution() {
+  return screen.width + DIMENSION_SEPARATOR + screen.height;
+}
+
+export function floorDimensionFields(field?: string | null) {
+  return !field
+    ? field
```

Comment:
Nit: Personally find `field && field.split()...` or flipping the conditionals a bit nicer here, starting with not for the ternary I find is a bit confusing.

Reply:
Yeap, missed that. Thanks and fixed!